python/lsst/task/base/task.py

```diff
@@ -1,6 +1,6 @@
 #
 # LSST Data Management System
-# Copyright 2008-2016 AURA/LSST.
+# Copyright 2008-2016 AURA/LSST; 2021 California Institute of Technology
```

Comment:
This copyright shouldn't be here at all. It should be referencing the COPYRIGHT file.

Reply:
Do we have a best-practice for what that looks like in file-header comments?  I looked in `lsst/templates` but it doesn't seem to have an obvious example.

Reply:
Ah: https://developer.lsst.io/python/style.html?highlight=copyright#each-python-file-must-contain-the-standard-license-preamble

Reply:
Fixed